pkg/kube/client.go

```diff
@@ -410,37 +411,41 @@ func newClientInternal(clientFactory *clientFactory, revision string) (*client,
 	if err != nil {
 		return nil, err
 	}
-	c.kubeInformer = informers.NewSharedInformerFactory(c.kube, resyncInterval)
+	c.kubeInformer = informers.NewSharedInformerFactoryWithOptions(c.kube, resyncInterval, informers.WithNamespace(features.InformerWatchNamespace))
```

Comment:
I guess if it's too complicated to have a list ( multiple informers, etc ) - we can keep it single namespace. Ignore my previous comment.